packages/next/server/config.ts

```diff
@@ -77,13 +95,31 @@ function assignDefaults(userConfig: { [key: string]: any }) {
         return currentConfig
       }
 
-      if (
-        key === 'experimental' &&
-        value !== defaultConfig[key] &&
-        typeof value === 'object' &&
-        Object.keys(value).length > 0
-      ) {
-        experimentalWarning(configFileName, Object.keys(value))
+      if (key === 'experimental' && typeof value === 'object') {
+        const notExistedExperimentalFeatures: string[] = []
+        const enabledExperimentalFeatures: (keyof ExperimentalConfig)[] = []
+
+        for (const featureName of Object.keys(
+          value
+        ) as (keyof ExperimentalConfig)[]) {
+          if (!(featureName in defaultConfig.experimental!)) {
+            notExistedExperimentalFeatures.push(featureName)
+          } else if (
+            value[featureName] !== defaultConfig.experimental![featureName]
+          ) {
+            enabledExperimentalFeatures.push(featureName)
+          }
+        }
+
+        if (notExistedExperimentalFeatures.length > 0) {
```

Comment:
I think `notExistedExperimentalFeatures` should be `enabledMissingExperiments`

You can also rename `enabledExperimentalFeatures` to `enabledExperiments`

Reply:
Fixed in [`a1927fe` (#37755)](https://github.com/vercel/next.js/pull/37755/commits/a1927fe8b6c6cf145e3402f95c1a8ae81ece95f5)